README.md

```diff
@@ -4,7 +4,7 @@ Maiko is the implementation of the Medley Interlisp virtual machine, for a
 byte-coded Lisp instruction set and some low-level functions for
 connecting with Lisp for access to display (via X11) and disk etc.
 
-For an overview, see [Medley Interlisp Introduction](https://github.com/Interlisp/medley/wiki/Medley-Interlisp-Introduction).
+For an overview, see [Medley Interlisp Introduction](https://github.com/Interlisp/medley/wiki/Introduction).
```

Comment:
We are in the process of deprecating the wiki page and centralizing all our documentation on our web site, [Interlisp.org](https://interlisp.org/).

A better place to link to may be:  [https://interlisp.org/medley/using/docs/medley/](https://interlisp.org/medley/using/docs/medley/)

The website is a work in progress - suggestions and improvements there are always appreciated.

Reply:
Thanks! Yes that's the page I was looking for :)